src/CMakeLists.txt

```diff
@@ -1,8 +1,32 @@
-add_library(sqlpp-mysql
-		connection.cpp
-		prepared_statement.cpp
-		char_result.cpp
-		bind_result.cpp
-		detail/connection_handle.cpp)
+add_library(sqlpp-mysql connection.cpp prepared_statement.cpp char_result.cpp
+                        bind_result.cpp detail/connection_handle.cpp)
 
 install(TARGETS sqlpp-mysql DESTINATION lib)
+install(TARGETS sqlpp-mysql EXPORT sqlpp11-connector-mysqlTargets)
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/cmake/sqlpp11-connector-mysqlConfigVersion.cmake"
+  VERSION ${TAG}
```

Comment:
If `TAG` is undefined, I believe this will be an error.

We call `write_basic_package_version_file` only in case that `TAG` is actually defined...